logging/src/test/java/org/mule/runtime/test/integration/logging/LogConfigurationTestCase.java

```diff
@@ -109,6 +118,43 @@ public void honorLog4jConfigFileForAppInDomain() throws Exception {
     ensureArtifactAppender("ConsoleForDomain", ConsoleAppender.class);
   }
 
+  @Test
+  @Feature(INTEGRATIONS_TESTS)
+  @Story(LOGGER)
+  @Issue("W-11090843")
+  public void honorLog4jConfigFileForTwoAppsWithDifferentConfiguration() throws Exception {
+    String customLogAppName = "custom-log-app";
+    String notCustomLogAppName = "not-custom-log-app";
+
+    String customAppLog = "Custom Log App log";
+    String notCustomAppLog = "Not Custom Log App log";
```

Comment:
use system properties for these and reference the system properties in the files